R/run_match_prioritize.R

```diff
@@ -2,15 +2,18 @@ run_match_prioritize <- function(config) {
   config <- load_config(config)
 
   dir_matched <- get_matched_dir(config)
-  path_abcd <- get_abcd_path(config)
-  sheet_abcd <- get_abcd_sheet(config)
+  abcd_dir <- get_abcd_dir(config)
 
   match_prio_priority <- get_match_priority(config)
 
   apply_sector_split <- get_apply_sector_split(config)
   sector_split_type_select <- get_sector_split_type(config)
 
   # validate config values----
+  stop_if_not_length(abcd_dir, 1L)
+  stop_if_not_inherits(abcd_dir, "character")
+  stop_if_dir_not_found(abcd_dir, desc = "ABCD data")
+
```

Comment:
maybe you want to check that the file exists in the specified directory (also below)?
```suggestion
  stop_if_dir_not_found(abcd_dir, desc = "ABCD data")
  stop_if_file_not_found(file.path(abcd_dir, desc = "ABCD final"))
```